turborepo-tests/helpers/setup_package_manager.sh

```diff
@@ -19,3 +19,17 @@ if [ "$pkgManager" != "" ]; then
 
   git commit -am "Updated package manager to $pkgManager" --quiet
 fi
+
+# If we're in a prysk test, set the corepack install directory to the prysk temp directory.
+# This will help isolate from the rest of the system, especially when running tests on a dev machine.
+if [ "$PRYSK_TEMP" == "" ]; then
+  COREPACK_INSTALL_DIR_CMD=
+else
+  COREPACK_INSTALL_DIR="${PRYSK_TEMP}/corepack"
+  mkdir -p "${COREPACK_INSTALL_DIR}"
+  export PATH=${COREPACK_INSTALL_DIR}:$PATH
+  COREPACK_INSTALL_DIR_CMD="--install-directory=${COREPACK_INSTALL_DIR}"
+fi
+
+# Enable corepack so that the packageManager setting in package.json is respected.
+corepack enable "${COREPACK_INSTALL_DIR_CMD}"
```

Comment:
I think we need this flag? I'm unsure reading the helptext for `corepack enable`
```suggestion
corepack enable --install-directory "${COREPACK_INSTALL_DIR_CMD}"
```

Reply:
The env var already includes this above (funky setup, but i just copied it over from the other script, so I didn't want to change it here)